### PR TITLE
Incorporate translations (received from BOSA) for the new strings in OAT

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-126

**Description**

As BOSA author
I would like the new functionality of Tabular Import to be provided with translations for French & Dutch
so that I can use it on my native language. 

**Acceptance Criteria** 

- AC1

```
GIVEN provided French translation for new keys introduced during Tabular Import feature implementation
   AND author is logged-in with preferred interface language set to French
THEN  TAO shall present Tabular Import related strings in French according to the translations provided
```

- AC2

```
GIVEN provided Dutch translation for new keys introduced during Tabular Import feature implementation
   AND author is logged-in with preferred interface language set to Dutch
THEN  TAO shall present Tabular Import related strings in Dutch according to the translations provided
```

**Environment**

http://aadvanced-search-integration.playground.kitchen.it.taocloud.org:48321

### CODE REVIEW

**Review by gabrielfs7 — APPROVED** [review]
Good job, @habemuscode ! :)

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: locales/fr-FR/messages.po [hunk 1]
```diff
@@ -1633,7 +1633,7 @@ msgid "You are currently working offline."
 msgstr ""
 
 msgid "You are encountering a prolonged connectivity loss. "
-msgstr "Vous rencontrez actuellement une perte de connectivité prolongée."
+msgstr "Vous rencontrez une perte de connectivité prolongée. "
```
Comment (jsconan) [inline locales/fr-FR/messages.po:1636]: **nitpick:** please avoid putting specific content in generic extension.
This is a customer specific flavour, hence it should go to the customer's extension

**Review by jsconan — CHANGES_REQUESTED** [review]
Please avoid disclosing any internal info into public repository.
Please update the PR description with something generic.
Edit: moreover, the description is not accurate with what the PR is doing...

I think the existing translations should not be updated with customer specific. Instead, it should go to the customer related extension.

Please also check this other PR that is doing the same with regard to translations:
https://github.com/oat-sa/extension-tao-testqti/pull/2026#pullrequestreview-674030051

**Comment by jsconan** [comment]
Actually, I think this one is redundant with https://oat-sa.atlassian.net/browse/TR-1251.
You may eventually close this PR as it introduces a regression. Sorry for the mess :/

**Comment by habemuscode** [comment]
Closing in favor of https://github.com/oat-sa/extension-tao-testqti/pull/2026

**Pull request closed without merging**